src/docs/codelabs/implicit-animations.md

```diff
@@ -62,10 +62,10 @@ Click the **Run** button to run the example:
 {% include implicit-animations/fade-in-starter-code.md %}
 
 {{site.alert.important}}
-  This page uses an embedded version of [DartPad] to display
+  This page uses an embedded version of [DartPad][] to display
```

Comment:
These fixes should probably be pulled out into a separate PR and actually landed

Reply:
That's what Andrew was doing. He just asked Kathy and I to test it. Closing this PR.